config/config.go

```diff
@@ -229,6 +230,26 @@ func newDefaultSectorbaseConfig() *SectorBaseConfig {
 	}
 }
 
+// SyncConfig holds all values that control how and when a filecoin node syncs its chain.
+type SyncConfig struct {
+	// CatchupSyncerPeriod represents how frequently a node will attempt to catch up when performing
+	// catchup sync.
+	CatchupSyncerPeriod string `json:"catchupSyncerPeriod"`
+	// TrustedAddress represents trusted peers to sync from.
+	TrustedAddresses []string `json:"trustedPeers"`
```

Comment:
Is it weird to have both `TrustedAddresses` here and `BootstrapAddresses` in the Bootstrap config?  It's not clear to me if these two fields are the same thing or not.

Reply:
Yes a bit weird as this change is premature. Currently the role of bootstrap nodes is overloaded, they mine blocks in addition to bootstrapping nodes to the network. In the future I believe we plan to separate these roles- bootstrappers and miners. Once that happens the addresses in this field will be different than the addresses in the bootstrap filed.